openpaas-james/pom.xml

```diff
@@ -706,6 +706,13 @@
                 <artifactId>feign-slf4j</artifactId>
                 <version>${feign.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.projectreactor</groupId>
+                <artifactId>reactor-bom</artifactId>
+                <version>2020.0.3</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
That solves the deployment, but still not the MemoryServerTest where I still get a connection reset (different issue).

What I don't understand is that we use exactly the same setup for integration tests and there is no issue on those...

Reply:
Ok it solves it it seems, in command line it's fine... My IDE seems quite buggy since it updated to the latest version... it affects my dev honestly. 

Will deal with the armored key instead of the base64 in the json next week...